webapp-django/crashstats/manage/admin.py

```diff
@@ -160,12 +160,11 @@ def graphics_devices(request):
     if request.method == 'POST' and 'file' in request.FILES:
         upload_form = forms.GraphicsDeviceUploadForm(request.POST, request.FILES)
         if upload_form.is_valid():
-            if upload_form.cleaned_data['database'] == 'pcidatabase.com':
-                function = utils.pcidatabase__parse_graphics_devices_iterable
-            else:
-                function = utils.pci_ids__parse_graphics_devices_iterable
+            devices = utils.pci_ids__parse_graphics_devices_iterable(
+                upload_form.cleaned_data['file']
+            )
 
-            for item in function(upload_form.cleaned_data['file']):
+            for item in devices:
```

Comment:
This is the interesting part of the change. Since we don't have two formats to support anymore, we can remove the dropdown from the form and the handling for it.